src/cfile/writer.py

```diff
@@ -493,7 +501,7 @@ def _write_function_usage(self, elem: core.Function) -> None:
         """
         Writes function usage (name of the function)
         """
-        if not elem.name:
+        if elem.name == "":
```

Comment:
Don't change this. It's making it worse.

Reply:
OK